th2_data_services/provider/v5/data_source/grpc.py

```diff
@@ -37,6 +37,12 @@
 
 
 class GRPCProvider5DataSource(IGRPCProviderDataSource):
+    """DataSource class which provide work with rpt-data-provider.
+
+    Rpt-data-provider version: 5.x.y
+    Protocol: GRPC
+    """
+
     def __init__(
```

Comment:
Really great work! But add, please, docs to GRPCProvider5DataSource.__init__ too.
Thanks.